packages/flame/lib/src/game/game.dart

```diff
@@ -363,3 +390,19 @@ abstract mixin class Game {
     gameStateListeners.forEach((callback) => callback());
   }
 }
+
+enum GameLifecycleState {
+  /// The game is backgrounded.
+  ///
+  /// This corresponds to [AppLifecycleState.paused] and [AppLifecycleState.detached].
+  backgrounded,
+
+  /// The game has been foregrounded but the first update should be
+  /// disregarded.
+  foregroundedFirstUpdate,
```

Comment:
Maybe just `foregrounding`? Should indicate clear enough that it is on its way to be in the foreground.